game/src/sandbox/dashboards/summaries.rs

```diff
@@ -55,14 +55,12 @@ impl TripSummaries {
             panel: Panel::new(Widget::col(vec![
                 DashTab::TripSummaries.picker(ctx, app),
                 Widget::row(vec![
-                    Widget::col(filters)
-                        .padding(16)
-                        .outline(ctx.style().section_outline),
+                    Widget::col(filters).section(ctx),
                     Widget::col(vec![
                         summary_boxes(ctx, app, &filter),
                         Widget::row(vec![
-                            contingency_table(ctx, app, &filter),
-                            scatter_plot(ctx, app, &filter),
+                            contingency_table(ctx, app, &filter).bg(ctx.style().section_bg),
```

Comment:
Unrelated to the tabs, but while I was here, I noticed a few things that should probably have "section" styling.